tests/components/zwave_js/test_climate.py

```diff
@@ -264,7 +264,7 @@ async def test_thermostat_v2(
     client.async_send_command.reset_mock()
 
     # Test setting invalid hvac mode
-    with pytest.raises(ValueError):
+    with pytest.raises(ServiceValidationError):
```

Comment:
There's dead code now:
https://github.com/home-assistant/core/blob/c359765a29c6013aed9eb12856021aadd9aa6337/homeassistant/components/zwave_js/climate.py#L495

I think we should clean up the integrations if we haven't done that already.